plugins/git/parser/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 include_directories(
   include
   ${PROJECT_SOURCE_DIR}/util/include
-  ${PROJECT_SOURCE_DIR}/parser/include)
+  ${PROJECT_SOURCE_DIR}/parser/include
+  ${PROJECT_SOURCE_DIR}/model/include)
```

Comment:
Why was this include added?

Reply:
It was added because some new includes (e.g. `model/File.h`) was added to `parsercontext.h` and therefore *gitparser* required the new include directory. Later `parsercontext.h` was separated into a header and body file to eliminate this added dependency, but it was left there. Good find!
Fixed in eaae9fdac3faadd46558e484a28a7930a32d3591.